qa/rpc-tests/mergetoaddress_helper.py

```diff
@@ -27,15 +27,14 @@ def assert_mergetoaddress_exception(expected_error_msg, merge_to_address_lambda)
 
 class MergeToAddressHelper:
 
-    def __init__(self, addr_type, any_zaddr, utxos_to_generate, utxos_in_tx1, utxos_in_tx2, test_mempooltxinputlimit):
+    def __init__(self, addr_type, any_zaddr, utxos_to_generate, utxos_in_tx1, utxos_in_tx2):
```

Comment:
There are a couple of places in this test where we say:
```
# Don't sync node 2 which rejects the tx due to its mempooltxinputlimit
sync_blocks(test.nodes[:2])
sync_mempools(test.nodes[:2])
test.nodes[1].generate(1)
test.sync_all()
```
These could be changed to
```
test.sync_all()
test.nodes[1].generate(1)
test.sync_all()
```
Now that we are no longer checking `mempooltxinputlimit`

Reply:
Yep, but that can go into a separate PR that fully-removes `mempooltxinputlimit` code.